src/components/Modal/Modal.test.tsx

```diff
@@ -202,6 +202,23 @@ describe('Modal component', () => {
     expect(modalWindow).not.toHaveAttribute('aria-describedby')
   })
 
+  it('throws an error if labelledby or describedby is null', async () => {
```

Comment:
Currently the codes is checking for `undefined` specifically (See comment on that). A test for null would be good as well though!

```suggestion
  it('throws an error if labelledby or describedby is undefined', async () => {
```